### ISSUE TITLE
rnv/template-starter@0.37.0-rc.14 issues running on ios device

### ISSUE THREAD

**Issue by GabrieleKaceviciute** [flexn-io/renative#1060]
**Describe the bug**
Rnv does not install app on iOS real device - iPhone 7 Plus, iOS 15.7.2

**To Reproduce**
1. Create new rnv project
2. Make sure that real ios device is connected
3.  Do `npx rnv run -p ios -d`

**Expected behavior**
App is built and installed to device

**Screenshots or copy&paste**
```
 $ rnv run -p ios -d                                                         │
├──────────────────────────────────────────────────────────────────────────────┤
│  ReNative Version: 0.37.0-rc.14                                               │
│  Project Name ($package.name): @rnv/template-starter                         │
│  Project Version ($package.version): 0.37.0-rc.14                             │
│  Workspace ($.workspaceID): rnv                                              │
│  Platform (-p): ios                                                          │
│  Engine ($.platforms.ios.engine): engine-rn                                  │
│  App Config (-c): template                                                   │
│  Build Scheme (-s): debug                                                    │
│  Target (-t): iPhone 14                                                      │
│  Supported Platforms: ios                                                    │
│  Env Info: darwin | x64 | node v16.14.2                                      │
│  Executed Time: 0h:0m:14s:174ms                                              │
│                                                                              │
│ [ success ] [run] Found one device connected! device name: iPhone (2) (15.7.2) udid: c1f91ab2e60ad3d1e25b3d70ec1272a173fa9488
│                                                                              │
│  Project location:                                                           │
│  ./platformBuilds/template_ios/RNVApp.xcworkspace                            │
└────────────────────────
```

**Desktop (please complete the following information):**

-   OS: macOS Monterey 12.6
-   Node Version: 16.4.2
-   RNV Version: 0.37.0-rc.14

**Comment by pauliusguzas** [comment]
On iPad (14.7.1) I am getting:
```
✖ FAILED: node /Users/pauliusguzas/Desktop/rnv14/node_modules/react-native/local-cli/cli.js run-ios --project-path platformBuilds/app_ios --scheme RNVApp --configuration Debug --udid 591253557404a06ec4a7ce4fc091c84bdcd76442 --allowProvisioningUpdates
┌──────────────────────────────────────────────────────────────────────────────┐
│  🚀  SUMMARY | 7/14/2023, 2:40:55 PM                                         │
│  $ rnv run -p ios -d -r                                                      │
├──────────────────────────────────────────────────────────────────────────────┤
│  ReNative Version: 0.37.0-rc.14                                              │
│  Project Name ($package.name): rnv14                                         │
│  Project Version ($package.version): 0.1.0                                   │
│  Workspace ($.workspaceID): rnv                                              │
│  Platform (-p): ios                                                          │
│  Engine ($.platforms.ios.engine): engine-rn                                  │
│  Template: @rnv/template-starter@0.37.0-rc.14                                │
│  App Config (-c): app                                                        │
│  Build Scheme (-s): debug                                                    │
│  Target (-t): iPhone 14                                                      │
│  Reset Project (-r): true                                                    │
│  Supported Platforms: ios                                                    │
│  Env Info: darwin | x64 | node v16.14.2                                      │
│  Executed Time: 0h:1m:8s:204ms                                               │
│                                                                              │
│ [ success ] [run] Found one device connected! device name: iPad (14.7.1) udid: 591253557404a06ec4a7ce4fc091c84bdcd76442
│ [ error ] [run] COMMAND:

node /Users/pauliusguzas/Desktop/rnv14/node_modules/react-native/local-cli/cli.js run-ios --project-path platformBuilds/app_ios --scheme RNVApp --configuration Debug --udid 591253557404a06ec4a7ce4fc091c84bdcd76442 --allowProvisioningUpdates

FAILED with ERROR:

error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening RNVApp.xcworkspace.
Command line invocation:
 /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace RNVApp.xcworkspace -configuration Debug -scheme RNVApp -destination id=591253557404a06ec4a7ce4fc091c84bdcd76442 -derivedDataPath build/RNVApp -allowProvisioningUpdates
User defaults from command line:
 IDEDerivedDataPathOverride = /Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/build/RNVApp
 IDEPackageSupportUseBuiltinSCM = YES
/Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/Pods/Pods.xcodeproj: error: Signing for "React-Core-AccessibilityResources" requires a development team. Select a development team in the Signing & Capabilities editor. (in target 'React-Core-AccessibilityResources' from project 'Pods')
2023-07-14 14:40:44.787 xcodebuild[46307:1294662] DVTAssertions: Warning in /System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot2/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/DVTiOSFrameworks/DVTiOSFrameworks-21506/DTDeviceKitBase/DTDKRemoteDeviceData.m:382
Details: (null) deviceType from 591253557404a06ec4a7ce4fc091c84bdcd76442 was NULL when -platform called.
Object: <DTDKMobileDeviceToken: 0x7fb170f24de0>
Method: -platform
Thread: <NSThread: 0x600002d307c0>{number = 7, name = (null)}

Command failed with exit code 1: node /Users/pauliusguzas/Desktop/rnv14/node_modules/react-native/local-cli/cli.js run-ios --project-path platformBuilds/app_ios --scheme RNVApp --configuration Debug --udid 591253557404a06ec4a7ce4fc091c84bdcd76442 --allowProvisioningUpdates
error React Native CLI uses autolinking for native dependencies, but the following modules are linked manually:
  - react-native-gesture-handler (to unlink run: "react-native unlink react-native-gesture-handler")
This is likely happening when upgrading React Native from below 0.60 to 0.60 or above. Going forward, you can unlink this dependency via "react-native unlink <dependency>" and it will be included in your app automatically. If a library isn't compatible with autolinking, disregard this message and notify the library maintainers.
Read more about autolinking: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md
- Building the app.........
- Building the app.........
- Building the app......
- Building the app.......
- Building the app.....
- Building the app......
- Building the app.
- Building the app...
- Building the app...
- Building the app....
- Building the app.......
- Building the app.
- Building the app...
- Building the app...
- Building the app..
- Building the app.......
- Building the app..
error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening RNVApp.xcworkspace.
info Found Xcode workspace "RNVApp.xcworkspace"
info Building (using "xcodebuild -workspace RNVApp.xcworkspace -configuration Debug -scheme RNVApp -destination id=591253557404a06ec4a7ce4fc091c84bdcd76442 -derivedDataPath build/RNVApp -allowProvisioningUpdates")
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace RNVApp.xcworkspace -configuration Debug -scheme RNVApp -destination id=591253557404a06ec4a7ce4fc091c84bdcd76442 -derivedDataPath build/RNVApp -allowProvisioningUpdates

User defaults from command line:
    IDEDerivedDataPathOverride = /Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/build/RNVApp
    IDEPackageSupportUseBuiltinSCM = YES

Prepare packages

Computing target dependency graph and provisioning inputs

Create build description
Build description signature: d60f2ed4e39c707bd0a2b51b5ed53af2
Build description path: /Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/build/RNVApp/Build/Intermediates.noindex/XCBuildData/d60f2ed4e39c707bd0a2b51b5ed53af2-desc.xcbuild

note: Building targets in dependency order
/Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/Pods/Pods.xcodeproj: error: Signing for "React-Core-AccessibilityResources" requires a development team. Select a development team in the Signing & Capabilities editor. (in target 'React-Core-AccessibilityResources' from project 'Pods')
/Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'RNGestureHandler' from project 'Pods')
/Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'fmt' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'fmt' from project 'Pods')
/Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'RCT-Folly' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'RCT-Folly' from project 'Pods')

2023-07-14 14:40:44.787 xcodebuild[46307:1294662]  DVTAssertions: Warning in /System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot2/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/DVTiOSFrameworks/DVTiOSFrameworks-21506/DTDeviceKitBase/DTDKRemoteDeviceData.m:382
Details:  (null) deviceType from 591253557404a06ec4a7ce4fc091c84bdcd76442 was NULL when -platform called.
Object:   <DTDKMobileDeviceToken: 0x7fb170f24de0>
Method:   -platform
Thread:   <NSThread: 0x600002d307c0>{number = 7, name = (null)}
Please file a bug at https://feedbackassistant.apple.com with this warning message and any useful information you can provide.
** BUILD FAILED **


info Run CLI with --verbose flag for more details.

error React Native CLI uses autolinking for native dependencies, but the following modules are linked manually:
  - react-native-gesture-handler (to unlink run: "react-native unlink react-native-gesture-handler")
This is likely happening when upgrading React Native from below 0.60 to 0.60 or above. Going forward, you can unlink this dependency via "react-native unlink <dependency>" and it will be included in your app automatically. If a library isn't compatible with autolinking, disregard this message and notify the library maintainers.
Read more about autolinking: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md
- Building the app.........
- Building the app.........
- Building the app......
- Building the app.......
- Building the app.....
- Building the app......
- Building the app.
- Building the app...
- Building the app...
- Building the app....
- Building the app.......
- Building the app.
- Building the app...
- Building the app...
- Building the app..
- Building the app.......
- Building the app..
error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening RNVApp.xcworkspace.

Error: Command failed with exit code 1: node /Users/pauliusguzas/Desktop/rnv14/node_modules/react-native/local-cli/cli.js run-ios --project-path platformBuilds/app_ios --scheme RNVApp --configuration Debug --udid 591253557404a06ec4a7ce4fc091c84bdcd76442 --allowProvisioningUpdates
error React Native CLI uses autolinking for native dependencies, but the following modules are linked manually:
  - react-native-gesture-handler (to unlink run: "react-native unlink react-native-gesture-handler")
This is likely happening when upgrading React Native from below 0.60 to 0.60 or above. Going forward, you can unlink this dependency via "react-native unlink <dependency>" and it will be included in your app automatically. If a library isn't compatible with autolinking, disregard this message and notify the library maintainers.
Read more about autolinking: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md
- Building the app.........
- Building the app.........
- Building the app......
- Building the app.......
- Building the app.....
- Building the app......
- Building the app.
- Building the app...
- Building the app...
- Building the app....
- Building the app.......
- Building the app.
- Building the app...
- Building the app...
- Building the app..
- Building the app.......
- Building the app..
error Failed to build iOS project. We ran "xcodebuild" command but it exited with error code 65. To debug build logs further, consider building your app with Xcode.app, by opening RNVApp.xcworkspace.
info Found Xcode workspace "RNVApp.xcworkspace"
info Building (using "xcodebuild -workspace RNVApp.xcworkspace -configuration Debug -scheme RNVApp -destination id=591253557404a06ec4a7ce4fc091c84bdcd76442 -derivedDataPath build/RNVApp -allowProvisioningUpdates")
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -workspace RNVApp.xcworkspace -configuration Debug -scheme RNVApp -destination id=591253557404a06ec4a7ce4fc091c84bdcd76442 -derivedDataPath build/RNVApp -allowProvisioningUpdates

User defaults from command line:
    IDEDerivedDataPathOverride = /Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/build/RNVApp
    IDEPackageSupportUseBuiltinSCM = YES

Prepare packages

Computing target dependency graph and provisioning inputs

Create build description
Build description signature: d60f2ed4e39c707bd0a2b51b5ed53af2
Build description path: /Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/build/RNVApp/Build/Intermediates.noindex/XCBuildData/d60f2ed4e39c707bd0a2b51b5ed53af2-desc.xcbuild

note: Building targets in dependency order
/Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/Pods/Pods.xcodeproj: error: Signing for "React-Core-AccessibilityResources" requires a development team. Select a development team in the Signing & Capabilities editor. (in target 'React-Core-AccessibilityResources' from project 'Pods')
/Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'RNGestureHandler' from project 'Pods')
/Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'fmt' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'fmt' from project 'Pods')
/Users/pauliusguzas/Desktop/rnv14/platformBuilds/app_ios/Pods/Pods.xcodeproj: warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 9.0, but the range of supported deployment target versions is 11.0 to 16.2.99. (in target 'RCT-Folly' from project 'Pods')
warning: Run script build phase 'Create Symlinks to Header Folders' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'RCT-Folly' from project 'Pods')

2023-07-14 14:40:44.787 xcodebuild[46307:1294662]  DVTAssertions: Warning in /System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot2/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/DVTiOSFrameworks/DVTiOSFrameworks-21506/DTDeviceKitBase/DTDKRemoteDeviceData.m:382
Details:  (null) deviceType from 591253557404a06ec4a7ce4fc091c84bdcd76442 was NULL when -platform called.
Object:   <DTDKMobileDeviceToken: 0x7fb170f24de0>
Method:   -platform
Thread:   <NSThread: 0x600002d307c0>{number = 7, name = (null)}
Please file a bug at https://feedbackassistant.apple.com with this warning message and any useful information you can provide.
** BUILD FAILED **


info Run CLI with --verbose flag for more details.
    at makeError (/Users/pauliusguzas/Desktop/rnv14/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/pauliusguzas/Desktop/rnv14/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)


│                                                                              │
│  Project location:                                                           │
│  ./platformBuilds/app_ios/RNVApp.xcworkspace                                 │
└──────────────────────────────────────────────────────────────────────────────┘
```

**Comment by ahwelgemoed** [comment]
getting the same, cant run TVOS, ios

**Comment by mirrorshades** [comment]
okay I got it working. here are a few things I had to do.

1. Make sure everything was installed and running with node v 16.
2. `cd platformBuilds/app_tvos` and then `pod install`
3. open platformBuilds/app_tvos in xcode and when you run the application you will see the annoying bitwise Yoga error. xcode suggested the quick fix.
4. clean the build and then when you run `rnv run` and pick tvos it worked for me. Hopefully it works for you too.
